topi/python/topi/nn/local_response_norm.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=invalid-name
+"""TVM operator for local response norm compute."""
+from __future__ import absolute_import
+import tvm
+from .pad import pad
+
+@tvm.target.generic_func
+def lrn_nchw(data, size, alpha=0.0001, beta=0.75, bias=2):
+    """Perform local response normalisation on the data
```

Comment:
We should write down the formula like in PyTorch (https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/normalization.py#L9-L36) and TensorFlow (https://www.tensorflow.org/api_docs/python/tf/nn/local_response_normalization).